client/client.js

```diff
@@ -16,6 +17,7 @@ class Client {
     }
 
     async start() {
+        await MonacoEditor.start(this, true /* overrideDefaultKeybindings */, { theme: 'vs-light' } /* monacoEditorConfig */);
```

Comment:
Wondering if it'd be worth providing an option for the students to switch between dark mode and light mode.